crates/swc/tests/fixture/issues-2xxx/2281/case1/output/index.js

```diff
@@ -1,4 +1,7 @@
-var _func = require("./src/func");
+Object.defineProperty(exports, "__esModule", {
```

Comment:
This test case have `noInterop: true`

Reply:
If there is nothing to export, should we omit the `__esModule` ?

Reply:
tsc always emit if the source is ESM module (noInterop: false)
But Babel will omit it if nothing to export.

Reply:
I will revert some commits to follow the Babel behaviour.